L126-core-add-alts-google-call-credentials.md

```diff
@@ -76,8 +78,10 @@ create GoogleDefaultCredentials by setting a GoogleDefaultCredentialsOptions
 value into their standard call. For this addition, the proposed struct
 `GoogleDefaultCredentialsOptions` will hold a boolean that will be default to
 false. Callers of the GoogleDefaultCredentials() API will be able to set
-use_alts to false value, if required to indicate the request for the underlying
-bound token call credentials.
+use_alts_call_credentials to false value, if required to indicate the request
+for the underlying bound token call credentials.
+
+### C++ Changes
```

Comment:
This heading needs to move up above the previous paragraph, which talks about the C++ API.